src/PaymentSuite/PaymillBundle/Resources/views/Paymill/scripts.html.twig

```diff
@@ -7,58 +7,96 @@
     /**
      * Validation for paymill payment
      */
-    $("#paymill_form").submit(function(){
 
-        $("#paymill_view_submit").attr("disabled", "disabled");
+    var oPaymill = {
```

Comment:
It's not necessary because the important values and actions are set on submit. I've tested in all browsers.